resources/aws/errors.go

```diff
@@ -13,6 +13,7 @@ var (
 
 	gatewayFindError       = errgo.New("Couldn't find gateway")
 	routeTableFindError    = errgo.New("Couldn't find route table")
+	routeFindError         = errgo.New("Couldn't find route")
```

Comment:
Rather minor. I am also a little late with my feedback/nitpicking here. We tend to define errors and their matchers like this: https://github.com/giantswarm/kubernetesd/blob/master/service/creator/error.go#L7. 

- Errors are private.
- Each error has its matcher. 
- Matchers are public. 
- Error messages start lower case. 
- Errors and matchers are grouped together.